testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -474,7 +474,7 @@ def compute_kiwi_profile_version(host)
   elsif host == 'sle15sp3_terminal'
     mac = $sle15sp3_terminal_mac
     mac = 'EE:EE:EE:00:00:06' if mac.nil?
-  elsif host.include? 'ubuntu'
+  elsif (host.include? 'dblike') || (host.include? 'debian11') || (host.include? 'ubuntu')
```

Comment:
Here why we don't abstract dblike to ubuntu or debian? Why all these options?

Reply:
CI test suite : generic `dblike`
BV test suite : specific `ubuntu1804`, `ubuntu2004`, `ubuntu2204`, `debian9`, `debian10`, `debian11`

Notice that `debian9` and `debian10` use normal `eth1`instead of `en4`...

Reply:
All good I think here, it's still the same